src/sdk/mini_cluster.h

```diff
@@ -55,7 +55,9 @@ namespace openmldb {
 namespace sdk {
 
 constexpr int MAX_TABLET_NUM = 3;
+#ifdef __linux__
```

Comment:
is mac broken ?

Reply:
Yes.  It fix bprc server core on mac . #247